fdbserver/storageserver.actor.cpp

```diff
@@ -4696,6 +4696,7 @@ ACTOR Future<Void> validateRangeAgainstServer(StorageServer* data,
 			req.limitBytes = limitBytes;
 			req.version = version;
 			req.tags = TagSet();
+			req.arena.dependsOn(range.arena());
```

Comment:
I think it would be best to do this after setting end or before setting begin so it's more clear why it is needed.

Reply:
Good point, done.